test/utils/utils_linux.cpp

```diff
@@ -169,3 +169,19 @@ TEST_F(test, utils_open) {
     EXPECT_EQ(utils_file_open(NULL), -1);
     EXPECT_EQ(utils_file_open_or_create(NULL), -1);
 }
+
+TEST_F(test, utils_align_ptr_up_size_down) {
+    uintptr_t ptr = 0x4000;
```

Comment:
Missing header for uintptr_t (`cstdint` or `stdint.h`).

Reply:
Done